src/UI/Component/Symbol/Glyph/Factory.php

```diff
@@ -951,4 +951,118 @@ public function time($action = null);
      * @return \ILIAS\UI\Component\Symbol\Glyph\Glyph
      */
     public function close($action = null);
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *      The More Glyph allows shortening a part of a set of entries that
+     *      are too long to be presented fully or would be overwhelming.
+     *      The More glyph offers viewing the rest of the shortened set of
+     *      entries so that the entire set becomes visible.
+     *   composition: >
+     *      The More Glyph uses the glyphicon-option-horizontal.
+     *   effect: >
+     *       Clicking the More Glyph shows the rest of the set of entries.
+     *   rivals:
+     *      Disclosure Glyph: >
+     *         The Disclosure Glyph hides the complete set of entries, wherear the
+     *         More Glyph only hides parts of it.
+     *      Mini-Action-Drop-Down: >
+     *         The in ListGUI without text is used to offer a
+     *         set of actions that cannot be displayed directly due to scarce space.
```

Comment:
Grammar? Why talking about a ListGUI which is not a KS concept? We have Standard Listing Panels using action drop downs.